src/openscad.cc

```diff
@@ -798,6 +766,12 @@ int main(int argc, char **argv)
 {
 	int rc = 0;
 	StackCheck::inst()->init();
+#ifdef OPENSCAD_QTGUI
+	const auto application_path = QCoreApplication::applicationDirPath().toLocal8Bit().constData();
```

Comment:
Not sure what happened here. `QCoreApplication::applicationDirPath()` will return an empty string until a `QCoreApplication` has been instantiated. Note: It doesn't matter _where_ this object is instantiated, as Qt will keep a pointer to it. Excerpt from the Qt source code:
```
QString QCoreApplication::applicationDirPath()
{
    if (!self) {
        qWarning("QCoreApplication::applicationDirPath: Please instantiate the QApplication object first");
        return QString();
    }
   [...]
```

Reply:
Oops, I must not have tested it after I changed it to a static call. I have reverted to instantiating an app and deleting it again.